examples/src/examples/AfiInteractiveOptimizerEngineExample.java

```diff
@@ -59,6 +55,9 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       afiOptimizerParam.setData(afiItem);
       
       ApiResponse<Object> response = apiInstance.postAndOptimizeWithHttpInfo(null, null, afiOptimizerParam);
+      //Comment the above line and uncomment the below lines to add cache control configuration. Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older.
+      //String cache_control="max-stale=300";
```

Comment:
Can you please use camel casing for `cache_control`?

Reply:
Updated the code with camel case.